integration_tests/README.md

```diff
@@ -476,6 +476,17 @@ properly without it. These tests assume Iceberg is not configured and are disabl
 If Spark has been configured to support Iceberg then these tests can be enabled by adding the
 `--iceberg` option to the command.
 
+### Run Apache iceberg s3tables tests
+
+To run iceberg tests against aws s3tables catalog, we need to setup several things:
+1. Run `aws configure` to setup aws credentials
+2. Create a s3tables [table bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-tables-buckets-create.html), and fetch its arn
+3. Create a namespace under the table bucket created in step 2.
```

Comment:
Add an example like?
```python
spark.sql(" CREATE NAMESPACE IF NOT EXISTS s3tablesbucket.my_namespace")
```

Is it possible to remove this step? make it automatic in the test case. This maybe a follow-up.